### PR TITLE
update default reply_delimiter

### LINKED ISSUE #218: default reply_delimiter
It looks like the default configuration setting for reply_delimiter is "Reply ABOVE THIS LINE" when the readme shows that the default is "-- REPLY ABOVE THIS LINE --". Should this be updated? If so, I can create a PR

### PR DESCRIPTION
**Pull request by dvanderbeek**
This makes the default setting "-- REPLY ABOVE THIS LINE --", which is consistent with the Readme and loooks better than "Reply ABOVE THIS LINE". Fixes #218.

### CODE REVIEW

File: spec/griddler/email_spec.rb [hunk 1]
```diff
@@ -57,7 +57,7 @@
 
   it 'handles everything on one line' do
     body = <<-EOF
-      Hello. On 01/12/13, Tristan <email@example.com> wrote: Reply ABOVE THIS LINE or visit your website to respond.
+      Hello. On 01/12/13, Tristan <email@example.com> wrote: -- REPLY ABOVE THIS LINE -- or visit your website to respond.
```
Comment (gabebw) [inline spec/griddler/email_spec.rb]: This is fine.

File: spec/griddler/email_spec.rb [hunk 2]
```diff
@@ -301,7 +301,7 @@
     expect(body_from_email(text: body)).to eq 'Hello.'
   end
 
-  it 'should remove any signature without space above Reply ABOVE THIS LINE' do
+  it 'should remove any signature without space above -- REPLY ABOVE THIS LINE --' do
```
Comment (gabebw) [inline spec/griddler/email_spec.rb]: This is fine.

**Comment by gabebw** [comment]
Thanks!

**Comment by dvanderbeek** [comment]
No problem. Thanks for creating the gem - it's really useful!

**Pull request merged**